### PR TITLE
fix tempest keystone_api

### PR DESCRIPTION
**Pull request by stuggi**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: ci_framework/roles/tempest/tasks/configure-tempest.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@
   ansible.builtin.set_fact:
     keystone_secret_name: "{{ keystone_data.stdout | from_json | community.general.json_query('spec.secret') }}"
     keystone_passwd_select: "{{ keystone_data.stdout | from_json | community.general.json_query('spec.passwordSelectors.admin') }}"
-    keystone_api: "{{ keystone_data.stdout | from_json | community.general.json_query('status.apiEndpoint.public') }}"
+    keystone_api: "{{ keystone_data.stdout | from_json | community.general.json_query('status.apiEndpoints.public') }}"
```
Comment (cjeanner) [inline ci_framework/roles/tempest/tasks/configure-tempest.yml:13]: did that ever work then? Or is it a new thing? Just wanting to understand the context here.

**Comment by cjeanner** [comment]
/approve

it seems it worked mostly "by chance" until now.

File: ci_framework/roles/tempest/tasks/configure-tempest.yml (hunk 1 above)
Comment (stuggi) [reply ci_framework/roles/tempest/tasks/configure-tempest.yml:13]: it always used the default for the endpoint

**Comment by stuggi** [comment]
` "error: unable to default to a user name: the server is currently unable to handle the request (get users.user.openshift.io ~)"`

**Comment by stuggi** [comment]
recheck

**Pull request merged**